adammathis05/dad-dinner
# Dad jokes modal- button is on results page and pops up empty modal

## Purpose

This PR adds a dad jokes button to the results page and when the button is clicked a modal with placeholder text pops up

Fixes #37 

## Changes

![image](https://github.com/adammathis05/dad-dinner/assets/169301676/548121b1-c5bb-41e5-ac8f-ce9cd1990506)

## Steps to Reproduce or Test

review script.js and results.html to view the code and view results .html in the browser to see and test the button

![image](https://github.com/adammathis05/dad-dinner/assets/169301676/b5ea6bb3-f4df-4139-a33d-eee8a6ba4fb0)

![image](https://github.com/adammathis05/dad-dinner/assets/169301676/64ead582-0117-41c0-b1d3-b1d57ad2e3c9)

## Optional GIF

Self-explanatory.
